### PR TITLE
Fix marker issue in winfsp read_directory operation

### LINKED ISSUE #1039: Extreme lenteur d'affichage (Explorateur W10 - sous-dossiers d'enclave contenant très grand nombre de docs
Après plusieurs minutes, la barre verte de progression n'a toujours pas disparu, et la roue tourne toujours. Il d'agit d'un sous-dossier d'une enclave avec un très grand nombre de fichiers pdf.

En cours ::
image

Après plusieurs minutes, pas de terminaison 
image

image

### PR DESCRIPTION
**Pull request by vxgmichel**
Fix #1039

### CODE REVIEW

**Comment by touilleMan** [comment]
newsfragment is missing though

**Comment by vxgmichel** [comment]
> newsfragment is missing though
Fixed!

File: tests/core/mountpoint/test_base.py [hunk 1]
```diff
@@ -523,3 +523,40 @@ def test_nested_rw_access(mountpoint_service):
...
+@pytest.mark.mountpoint
+@pytest.mark.parametrize("n", [10, 100, 1000])
+@pytest.mark.parametrize("base_path", ["/", "/foo"])
+async def test_mountpoint_iterdir_with_many_files(
+    n, base_path, base_mountpoint, alice_user_fs, event_bus
+):
+    wid = await alice_user_fs.workspace_create("w")
+    workspace = alice_user_fs.get_workspace(wid)
+    await workspace.mkdir(base_path, parents=True, exist_ok=True)
+    names = [f"some_file_{i:03d}.txt" for i in range(n)]
```
Comment (touilleMan) [inline tests/core/mountpoint/test_base.py]: What's the need of having multiple values for `n` ?

Comment (vxgmichel) [reply tests/core/mountpoint/test_base.py]: Typically, the problem fixed in this PR appears with more than ~100 files in a non-root directory. I felt the best way to cover those weird cases is to parametrize. The parameters can be adjusted over time.

Comment (touilleMan) [reply tests/core/mountpoint/test_base.py]: 👍

**Pull request merged**